Documentation/nvme-tls-key.txt

```diff
@@ -54,15 +54,73 @@ OPTIONS
 -e::
 --export::
 	Write the key data to the file specified by '--keyfile'
-	or stdou if not present.
+	or stdout if not present.
+
+-r <id>::
+--remove <id>::
```

Comment:
`=` needed and is `<id>` correct since seems better to be `<name>`, `<key>`, `<keyring>`, `<key-name>` or `<keyring-name>`.
```diff
- --remove <id>::
+ --remove=<name>::
```

Reply:
Yep, tt's `desc` after reading a bit more on the keyring interface.